cockroach/prisma_init.sql

```diff
@@ -1,17 +1,44 @@
 CREATE USER prisma;
 GRANT admin TO prisma;
 
+-- https://www.cockroachlabs.com/docs/stable/cluster-settings.html
+
+-- ???
```

Comment:
this means `CREATE TABLE t (a INT)` is INT4 (PG default), not INT8 (CRDB default)

Reply:
```suggestion
-- `CREATE TABLE t (a INT)` is INT4 (PG default), not INT8 (CRDB default
```